pkg/splunk/enterprise/util.go

```diff
@@ -736,6 +786,119 @@ func AddOrUpdateAppSrcDeploymentInfoList(appSrcDeploymentInfo *enterprisev1.AppS
 			newAppInfoList = append(newAppInfoList, appDeployInfo)
 			appChangesDetected = true
 		}
+
+		// Did an app change?  Let's do something about it... vvvv-Maybe remove this check-vvvv or make it "if pod is ready"
+		if appChangesDetected == true && prevDeployStatus == enterprisev1.DeployStatusComplete {
```

Comment:
This check isn't right either.  It will start trying to download/install apps prior to the pod/container being ready and simply fail.  Ideally we should wait until after the first run/initialization of splunk before we start running this code on appSrc changes.